Sources/Profile-View.php

```diff
@@ -1299,8 +1297,7 @@ function list_getNumUnwatched($memID)
 	$request = $smcFunc['db_query']('', '
 		SELECT COUNT(*)
 		FROM {db_prefix}log_topics as lt
-		LEFT JOIN {db_prefix}topics as t ON (lt.id_topic = t.id_topic)
-		LEFT JOIN {db_prefix}boards as b ON (t.id_board = b.id_board)
+		LEFT JOIN {db_prefix}topics as b ON (lt.id_topic = b.id_topic)
```

Comment:
Once `query_see_topic_board` is added, the query here can go back to using `{db_prefix}topics AS t` along with `{query_see_topic_board}`